src/database/index.tsx

```diff
@@ -41,6 +42,9 @@ const scenariosCollectionId = "scenarios";
 const facilitiesCollectionId = "facilities";
 const modelVersionCollectionId = "modelVersions";
 const usersCollectionId = "users";
+// TODO: when the datasource stabilizes, change this to the real collection
```

Comment:
For TODOs that will not be resolved in the current PR, let's always try to put an issue number with them, e.g. `TODO(XXX): When the datasource stabilizes...`